src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -184,9 +184,12 @@ public void getIndexNumber(String[] commandComponents) {
         }
     }
 
-    public void getSearchString(String[] commandComponents) {
+    public void constructSearchString(String[] commandComponents) {
         try {
-            searchString = commandComponents[SEARCH_STR_INDEX];
+            searchString = commandComponents[SEARCH_STR_INDEX_START];
+            for (int i = SEARCH_STR_INDEX_START + 1; i < commandComponents.length; i++) {
+                searchString += SEPARATOR + commandComponents[i];
+            }
         } catch (IndexOutOfBoundsException e) {
             throw new InvalidEmptyKeywordError();

```

Comment:
Can rename this error method

Reply:
Renamed to InvalidEmptySearchStringError